test/unit/bigip/test_bigip_asm_policy.py

```diff
@@ -0,0 +1,580 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017 F5 Networks Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import os
+import json
+import pytest
+import sys
+
+from nose.plugins.skip import SkipTest
+if sys.version_info < (2, 7):
+    raise SkipTest("F5 Ansible modules require Python >= 2.7")
+
+from ansible.compat.tests import unittest
+from ansible.compat.tests.mock import patch, Mock, PropertyMock
+from ansible.module_utils import basic
+from ansible.module_utils._text import to_bytes
+from ansible.module_utils.f5_utils import AnsibleF5Client
+from ansible.module_utils.f5_utils import F5ModuleError
+
+try:
+    from library.bigip_asm_policy import Parameters
+    from library.bigip_asm_policy import ModuleManager
+    from library.bigip_asm_policy import ArgumentSpec
+except ImportError:
+    try:
+        from ansible.modules.network.f5.bigip_asm_policy import Parameters
+        from ansible.modules.network.f5.bigip_asm_policy import ModuleManager
+        from ansible.modules.network.f5.bigip_asm_policy import ArgumentSpec
```

Comment:
iControlUnexpectedHTTPError needs to be imported here. see below for an example

https://github.com/F5Networks/f5-ansible/blob/devel/test/unit/bigip/test_bigip_software.py#L45